hnn_core/tests/test_cells_default.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 
 from neuron import h
+from numpy import allclose, abs
```

Comment:
```suggestion
import numpy as np
```

more standard

Reply:
the other issue is that you are overriding the `abs` that comes natively from python that may lead to unexpected issues.

Reply:
Whoops, bad form...

Reply:
fix this and I'll merge :)